src/server/db_slice.h

```diff
@@ -312,6 +312,10 @@ class DbSlice {
   // Unregisted all watched key entries for connection.
   void UnregisterConnectionWatches(ConnectionState::ExecInfo* exec_info);
 
+  void SetExpireAllowed(bool is_allowed) {
+    expire_allowed_ = is_allowed;
```

Comment:
you did not push the db_slice.cc changes

Reply:
I just added it in the header, no source changes.

Reply:
where is expire_allowed_ used? I see you just set it

Reply:
You're right :see_no_evil: thanks!